pipelines/rj_cvl__osinfo_pdf_reconstruct/tasks.py

```
@@ -0,0 +1,291 @@
+            print(f"  - Missing EOF marker")
+
+    return validation
+
+
+@task
+def save_pdf_to_gcs(
+    file_id: str,
+    pdf_data: bytes,
+    bucket_name: str = "rj-iplanrio",
+    prefix: str = "staging/brutos_osinfo_mongo/files_pdfs",
+    filename: str = None
```

Ruff (0.14.8)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)